### PR TITLE
DOC missing requirements for html doc build

### PR DESCRIPTION
**Pull request by soupault**
In accordance to https://github.com/hmmlearn/hmmlearn/issues/55 .

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Building the docs
 
 To build the docs you need to have the following packages installed::
 
-    $ pip install Pillow matplotlib Sphinx numpydoc
+    $ pip install Pillow matplotlib Sphinx sphinx-gallery sphinx_rtd_theme numpydoc
```
Comment (superbobry) [inline README.rst]: Are you sure `sphinx_rtd_theme` is required?

Comment (soupault) [inline README.rst]: `make` was looking for it on win64, yes. On Arhclinux it's set as a requirement for `python-sphinx`.

Will double check and update you tomorrow.

**Comment by soupault** [comment]
I suppose this should confirm the requirement:

    Successfully uninstalled sphinx-rtd-theme-0.1.9

    C:\Workspace\hmmlearn\doc>make
    sphinx-build -D plot_gallery=False -b html -d _build/doctrees   . _build/html
    Running Sphinx v1.3.1

    Exception occurred:
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\theming.py", line 30, in <module>
        import sphinx_rtd_theme
    ImportError: No module named 'sphinx_rtd_theme'
    The full traceback has been saved in C:\Users\{User}\AppData\Local\Temp\sphinx-err-4fd6opjz.log, if you want to report the issue to the developers.
    Please also report this if it was a user error, so that a better error message can be provided next time.
    A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
    Makefile:48: recipe for target 'html-noplot' failed
    make: *** [html-noplot] Error 1

    C:\Workspace\hmmlearn\doc>

Here is the file `C:\Users\{User}\AppData\Local\Temp\sphinx-err-4fd6opjz.log` with traceback :

    # Sphinx version: 1.3.1
    # Python version: 3.4.3 (CPython)
    # Docutils version: 0.12 release
    # Jinja2 version: 2.8
    # Last messages:

    # Loaded extensions:
    Traceback (most recent call last):
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\cmdline.py", line 244, in main
        opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\application.py", line 143, in __init__
        self.setup_extension(extension)
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\application.py", line 440, in setup_extension
        ext_meta = mod.setup(self)
      File "C:\Anaconda3\lib\site-packages\sphinxgallery\gen_gallery.py", line 95, in setup
        app.add_stylesheet('gallery.css')
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\application.py", line 686, in add_stylesheet
        from sphinx.builders.html import StandaloneHTMLBuilder
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\builders\html.py", line 37, in <module>
        from sphinx.theming import Theme
      File "C:\Anaconda3\lib\site-packages\Sphinx-1.3.1-py3.4.egg\sphinx\theming.py", line 30, in <module>
        import sphinx_rtd_theme
    ImportError: No module named 'sphinx_rtd_theme'

**Comment by superbobry** [comment]
Thanks, this looks convincing.

**Pull request merged**